components/organisms/TopicBox.js

```diff
@@ -65,4 +71,8 @@ TopicBox.propTypes = {
    * Test id for cypress test
    */
   dataCy: PropTypes.string,
+  /**
+   * array of subtopics to be listed
+   */
+  subtopics: PropTypes.array.string,
```

Comment:
`PropTypes.array.string` means that it's an array containing strings, right?

Reply:
it works like this, but further research tells me i probably dont need the string portion